sebs/gcp/gcp.py

```
@@ -239,246 +143,1429 @@ def _execute_with_retry(
+    @staticmethod
+    def function_type() -> "Type[Function]":
+        """Get the function class type for this platform.
+
+        Returns:
+            GCPFunction class type
+        """
+        return GCPFunction
+
+    def initialize(
+        self,
+        config: Dict[str, str] = {},
```

Ruff (0.15.12)

[warning] Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)